core/common/src/main/java/alluxio/worker/dora/DoraWorker.java

```diff
@@ -51,6 +51,16 @@ public interface DoraWorker extends DataWorker, SessionCleanable {
   @Nullable
   UfsStatus[] listStatus(String path, ListOptions options) throws IOException;
 
+  /**
+   * Invalidate all cached pages of this file.
+   *
+   * @param fileInfo the FileInfo of this file. Cached pages are identified by PageId and PageId is
+   *                 generated from fileInfo.fullUfsPath.
```

Comment:
space?

Reply:
Intentionally aligned with the first word after the param name.